### ISSUE TITLE
Possible stale cache of hybrid objects

### ISSUE THREAD

**Issue by khenderick** [openvstorage/framework#520]
Possible stale cache of hybrid objects. Following was observed (please note that there's only one vpool on that env)

```
from ovs.dal.lists.vpoollist import VPoolList
from ovs.dal.hybrids.vpool import VPool
a = VPoolList.get_vpools()[0]
print a._data['_version']  # prints 3
b = VPool(a.guid)
print b._data['_version']  # prints 2
```

**Comment by khenderick** [comment]
Fixed by #744

**Comment by JeffreyDevloo** [comment]
Some more information around this issue that I got when talking to khenderick: 
- A one time error
- Occurred on the creation of a vpool
- Can occur when both save and load happen in a small time window

In the last three weeks I've setup over a hundred vpools without ever bumping into this issue.
After reviewing the code I can state that the rate of encountering this one once more is nullable.
We plan to integrate a test that will check the staleness of every DAL object and I will post back when we encountered another one.
Closing.
